packages/fxa-auth-server/tsconfig.json

```diff
@@ -14,11 +14,9 @@
     "allowSyntheticDefaultImports": true,
     "noImplicitReturns": true,
     "noFallthroughCasesInSwitch": true,
-    "types": [
-      "node"
-    ],
     "typeRoots": [
       "./types",
+      "../fxa-shared/types",
```

Comment:
If at some point in the future, we want typeRoots to diverge based on building for deploy vs. running locally, we can make another tsconfig.prod.json that overrides just the bits to change when we build for it.